pyteal/ast/router.py

```diff
@@ -281,6 +287,9 @@ class CondWithMethod:
     method: ABIReturnSubroutine
 
     def to_cond_node(self, use_frame_pt: bool = False) -> CondNode:
+        if not (isinstance(self.condition, Expr) or self.condition == 1):
+            raise TealInputError("Invalid condition input for CondWithMethod")
+
```

Comment:
@ahangsu - this may be a result of a bad merge. Should I remove this assertion?

Reply:
I think it is a dup, we should take them away
```suggestion
```